ldes-server-port-fetch-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/rest/converters/LdesFragmentHttpConverter.java

```diff
@@ -40,7 +40,7 @@ public boolean canWrite(Class<?> clazz, MediaType mediaType) {
 
     @Override
     public List<MediaType> getSupportedMediaTypes() {
-        return List.of(new MediaType("application/ld+json"), new MediaType("application/n-quads"), new MediaType("application/turtle"));
+        return List.of(new MediaType("application/turtle"), new MediaType("application/ld+json"), new MediaType("application/n-quads"));
```

Comment:
forgot to update this. since we now support all formats that Jena can handle, this should be removed

Reply:
Let's handle in separate PR.